.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
 inherit_gem:
   gc_ruboconfig: rubocop.yml
+require: rubocop-rails
```

Comment:
For the cop `Rails/Blank:`

Reply:
This isn't a rails service why are we using the rails lint?

Reply:
That's correct, instead of just using `rubcop`, we use `gc_ruboconfig` which uses `rubocop-rails` thus the rails cops are enabled here!
https://github.com/gocardless/ibandit/blob/f44aa98457e16706c61800b7d3a6b7dd47796d8f/ibandit.gemspec#L6

Reply:
Right but should we? Also if we wanted to get the gc rails rubocop it should be like this

```
inherit_gem:
  gc_ruboconfig:
    - rubocop.yml
    - rails.yml
```

see https://github.com/gocardless/gc_ruboconfig/releases/tag/v3.0.0